src/OpenApi/Paths.hs

```diff
@@ -72,8 +72,7 @@ data Segment = Static Text | Dynamic Text
   deriving stock (Eq, Ord, Show)
 
 newtype Template = Template [Segment]
-  deriving newtype (Eq, Ord)
-  deriving stock   Show
+  deriving stock (Eq, Ord, Show)
```

Comment:
I think the `Ord` move should be in it's own commit. There are a few other types with `Ord` in this change, and they could be done together.